generate/testdata/queries/schema.graphql

```diff
@@ -111,6 +111,7 @@ type Query {
   getJunk: Junk
   getComplexJunk: ComplexJunk
   listOfListsOfLists: [[[String!]!]!]!
+  listOfListsOfListsOfContent: [[[Content!]!]!]!
```

Comment:
`listOfListsOfListsOfContent: [[[Content]!]!]!` is a type where one might want to use a pointer (though types like this usually seem to usually be accidental). Pointers to slices shouldn't be necessary, so it seems correct to only support pointers on the most-inner type.

Reply:
I think there are valid uses for required-list-of-optional-item, if you want the input in correspondence to the output.  But I agree I can only see a reason to support it at the innermost point.